### PR TITLE
[pt-BR] A lots of updates

### PR DESCRIPTION
**Pull request by marceloboeira**
Tons of translations, by @gabrielalmir10 :+1:

### CODE REVIEW

**Comment by irlnathan** [comment]
@marceloboeira @gabrielalmir10 Many thanks!

**Comment by irlnathan** [comment]
@gabrielalmir10 @marceloboeira I also added you as contributors to the Portuguese Translation Team.

**Comment by marceloboeira** [comment]
@irlnathan thanks a lot!

File: README.md [hunk 1]
```diff
@@ -1,16 +1,14 @@
-# Sails.js Documentation (v0.10.x)
+# Documentação Sails.js (v0.10.x)
```
Comment (dudemelo) [inline README.md]: Sails 0.10.x ?

**Comment by marceloboeira** [comment]
@gabrielalmir10

**Comment by gabrielalmir10** [comment]
@irlnathan Thank you :smile:

**Pull request merged**